Misc/NEWS.d/next/Core and Builtins/2018-04-26-22-39-17.bpo-33354.g35-44.rst

```diff
@@ -0,0 +1,2 @@
+Skip `test_load_dh_params` when Python filesystem encoding cannot encode the
```

Comment:
Please move this NEWS entry to the Tests section, and mention "test_ssl".

Reply:
Sorry, I forgot to do the same with this NEWS entry. Corrected in my last commit.